src/components/MediaSettings/MediaSettings.vue

```diff
@@ -399,6 +403,8 @@ export default {
 
 		closeModal() {
 			this.modal = false
+			this.updatedBackground = undefined
+			this.deviceIdChanged = false
 		},
 
 		toggleAudio() {
```

Comment:
Small wish: Audio/Video toggle buttons here in TopBar are not syncing (although they could with the BrowserStorage).
How about:
1) on `toggleAudio` and `toggleVideo` also add a flag `this.deviceStateChanged = true` and follow it for 'Apply Changes` button
2) on LocalAudioControlButton (and LocalVideoControlButton) add `subscribe('talk:media-settings:hide', this.updateDeviceState)`
```
updateDeviceState() {
if (BrowserStorage.getItem('audioDisabled_' + this.token)) {
  this.model.disableAudio()
} else {
  this.model.enableAudio()
}
}
```
3) also for LocalAudioControlButton expose method `toggleAudio` and replace in TopBarMediaControls duplicated method with `this.$refs.audioControl.toggelAudio()`